sdkcompat/v223/com/google/idea/sdkcompat/scala/ScalaCompat.java

```diff
@@ -0,0 +1,22 @@
+package com.google.idea.sdkcompat.scala;
+
+import com.intellij.psi.PsiMethod;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScObject;
+import org.jetbrains.plugins.scala.util.ScalaMainMethodUtil;
+import scala.Option;
+
+/** Provides SDK compatibility shims for Scala classes, available to IntelliJ CE & UE. */
+public class ScalaCompat {
```

Comment:
Is this necessary after all? It seemed to me that all the ScalaCompat for recent versions were identical